doc/spire_agent.md

```diff
@@ -345,13 +345,13 @@ agent {
 SPIRE agent has support for the [Envoy](https://envoyproxy.io) [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret) (SDS).
 SDS is served over the same Unix domain socket as the Workload API. Envoy processes connecting to SDS are attested as workloads.
 
-[`auth.TlsCertificate`](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/cert.proto#envoy-api-msg-auth-tlscertificate)
```

Comment:
Should I update `auth.TlsCertificate` to be `tls.TlsCertificate` or `transport_socket.TlsCertificate`?

Reply:
I think `tls.TlsCertificate` or `tlsv3.TlsCertificate` is the right call (tlsv3 is the package name for golang)